stdlib/LinearAlgebra/test/cholesky.jl

```diff
@@ -263,7 +261,6 @@ end
     for A in (R, C)
         @test !LinearAlgebra.issuccess(cholesky(A))
         @test !LinearAlgebra.issuccess(cholesky!(copy(A)))
-        @test_throws PosDefException LinearAlgebra.chol!(copy(A))
```

Comment:
(Likewise here, this straggler generated a depwarn, and is now redundant with the test a line above.)